index.html

```diff
@@ -272,6 +48,77 @@ <h4>
             </div>
         </div>
     </header>
+    <section class="choose_us">
+        <div class="container choose_us_container">
+            <div class="choose_us_main">
+                <h2 class="choose_us_h2">
```

Comment:
specifying tag name inside class name is not the best idea as it limits you to use it only with the mentioned tag. Otherwise it will look weird. What if you want to use same class for another tag? Creating a new class with the same styles is wrong. So I suggest you to name it `choose_us_title` instead of `choose_us_h2`. 
I understand that in this situation you will use this class only once and my suggestion doesn't make much sense. But when you start creating the next blocks you will see that all titles are the same  and you could use 1 commons class for all of them.